config/locales/application_form.yml

```
@@ -818,6 +818,33 @@ en:
               in_future: Enter a year before %{date}
             other_grade:
               blank: Enter your grade
+        candidate_interface/science_gcse_grade_form:
+          attributes:
+            single_award_grade:
+              blank: Enter your science single award grade
+              invalid: Enter a real science single award grade
+            double_award_grade:
+              blank: Enter your science double award grade
+              invalid: Enter a real science double award grade
+            grade:
+              blank: Enter your science grade
+              invalid: Enter a real science grade
+              too_long: Grade must be %{count} characters or fewer
+            biology_grade:
+              invalid: Enter a real Biology grade
+              blank: Enter your Biology grade
+            chemistry_grade:
+              invalid: Enter a real Chemistry grade
+              blank: Enter your Chemistry grade
+            physics_grade:
+              invalid: Enter a real Physics grade
+              blank: Enter your Physics grade
```

We only capitalise `English` subject name as it is a proper noun (or so the content gurus tell me)

```suggestion
            biology_grade:
              invalid: Enter a real biology grade
              blank: Enter your biology grade
            chemistry_grade:
              invalid: Enter a real chemistry grade
              blank: Enter your chemistry grade
            physics_grade:
              invalid: Enter a real physics grade
              blank: Enter your physics grade
```

---

👍